arch/arm/core/cortex_m/mpu/arm_mpu_v7_internal.h

```diff
@@ -106,6 +106,16 @@ static inline void _get_mpu_ram_region_attr(arm_mpu_region_attr_t *p_attr,
 	p_attr->rasr = _get_region_attr(1, ap, 1, 1, 1, 1, 0, size);
 }
 
+#ifdef CONFIG_RAM_FUNCTION
+static inline void _get_mpu_ramfunc_region_attr(arm_mpu_region_attr_t *p_attr,
+						u32_t base, u32_t size)
+{
+	ARG_UNUSED(base);
+
+	p_attr->rasr = _get_region_attr(0, RO, 0, 1, 0, 0, 0, size);
+}
+#endif
```

Comment:
Add /* xxx */ inline

Reply:
means just leave a /* xxx */ comments here?

Reply:
I mean, /* CONFIG_RAM_FUNCTION */ or whatever will be the name of the Kconfig option, after the review.

Reply:
And  I meant, on line 118: #endif /* CONFIG_RAM_FUMCTION */
(so it is clear what this #endif refers to)

Sorry for the confusion